src/editor/Model/Oni_Model.re

```diff
@@ -23,4 +23,5 @@ module StatusBarModel = StatusBarModel;
 module State = State;
 module SyntaxHighlighting = SyntaxHighlighting;
 module Tokenizer = Tokenizer;
+module Tokenizer2 = Tokenizer2;
```

Comment:
Rename `Tokenizer2` -> `Tokenizer`, and rename the old `Tokenizer` to `BufferViewTokenizer`.